enterprise/signals/parameter.py

```diff
@@ -233,6 +261,17 @@ def NormalSampler(mu, sigma, size=None):
         return np.random.normal(mu, sigma, size=size)
 
 
+def NormalPPF(value, mu, sigma):
+    """Prior function for Normal parameters.
+    Handles scalar mu and sigma, compatible vector value/mu/sigma,
+    vector value/mu and compatible covariance matrix sigma."""
+
+    if np.ndim(sigma) == 2:
```

Comment:
Should this be >= 2? It looks like this ppf is for a 1D Gaussian, and the class factory for Normal can take more than 2 values for mu, sigma. Alternatively, we could implement a multivariate Gaussian ppf.

Reply:
Ah! I totally missed this comment. Yes, good point, I made the change. PPF for multivariate Gaussian is indeed possible,  but let's postpone that for now